content/_policy/accessibility._en.md

```diff
@@ -25,8 +25,9 @@ Login.gov is regularly evaluated through a series of semi-automated evaluation t
 
 Our completed accessibility assessment reports can be found below:
 
-[IAL 1 Account creation report](/docs/accessibility-assessment-ial1-account-creation.pdf)
-[Identity verification report](/docs/accessibility-assessment-identity-verification-process.pdf)
+* [IAL 1 Account creation report](/docs/accessibility-assessment-ial1-account-creation.pdf)
```

Comment:
Not part of this ticket but should we do the same thing with the IAL1 account creation report as well.

Reply:
Agreed. I can make a ticket for this work.